helm/inteli-api/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: inteli-api
-appVersion: '1.20.3'
+appVersion: '1.23.0'
```

Comment:
why the big version bump?

Reply:
I was on 22 then a conflict merge appeared which all the way back down. I'll try and make this mystery number game happen at a lower level.